<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wordbook</class>
 <widget class="QDialog" name="wordbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR DCHYJianWeiDGB</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>白词斩</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR DCHYJianWeiDGB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>单词本</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>550</width>
     <height>400</height>
    </rect>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>160</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>310</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>490</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>600</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>600</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
